Header header
float32[]  sample
float32[]  max_val
float32[]  ratio
float32[]  slope
float64[]  voltage